tutorials/00_getting_started.ipynb

```diff
@@ -353,21 +343,47 @@
       "text/plain": [
-       "Drawing 10000 posterior samples:   0%|          | 0/10000 [00:00<?, ?it/s]"
+       "<Figure size 600x600 with 9 Axes>"
       ]
      },
      "metadata": {},
      "output_type": "display_data"
-    },
+    }
+   ],
+   "source": [
+    "samples = posterior.sample((10000,), x=x_obs)\n",
+    "pairplot(samples,\n",
+    "         points=theta_true,\n",
+    "         limits=[[-2, 2], [-2, 2], [-2, 2]], figsize=(6, 6),\n",
+    "         labels=[r\"$\\theta_1$\", r\"$\\theta_2$\", r\"$\\theta_3$\"]);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assessing the predictive performance of the posterior\n",
+    "\n",
+    "Another way to assess the quality of the posterior is checking whether parameters\n",
+    "sampled from the posterior $p(\\theta \\mid x_{obs})$ can reproduce the conditioned\n",
+    "observation $x_{obs}$ when we simulate data with them. This *posterior predictive\n",
+    "distribution* should be centered on the $x_{obs}$. We can again use the `pairplot`\n",
+    "function to visualize it. \n",
+    "\n",
+    "As you can see below, in this case the posterior predictive distribution is nicely\n",
+    "centered on the data it was conditioned on."
```

Comment:
again, it does not have to be centered

Reply:
here, it's a fact that it is centered on it because it's a Gaussian example. How about 

```
As you can see below, in this Gaussian toy example, the posterior predictive
distribution is nicely centered on the data it was conditioned on.
```

Reply:
Good point! Yes that works!